src/commands/addons/config.js

```diff
@@ -1,6 +1,4 @@
 const Command = require('../../utils/command')
-const { getAddons, updateAddon } = require('netlify/src/addons')
```

Comment:
Nice removal of `require('netlify/src/addons')` :tada:

Reply:
I'm not aware of anywhere we're using that code, so I believe the answer it yes  🎉

Reply:
We should open an issue for this. That's great news!

Reply:
https://github.com/netlify/js-client/issues/173